optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/api/PlannerBenchmarkFactory.java

```diff
@@ -471,17 +471,6 @@ public static PlannerBenchmarkFactory create(PlannerBenchmarkConfig benchmarkCon
         return new DefaultPlannerBenchmarkFactory(benchmarkConfig);
     }
 
-    /**
-     * As defined by {@link #create(PlannerBenchmarkConfig)}.
-     * @param benchmarkConfig never null
-     * @param classLoader sometimes null, the {@link ClassLoader} to use for loading all resources and {@link Class}es,
-     * null to use the default {@link ClassLoader}
-     * @return never null
-     */
-    public static PlannerBenchmarkFactory create(PlannerBenchmarkConfig benchmarkConfig, ClassLoader classLoader) {
-        return new DefaultPlannerBenchmarkFactory(benchmarkConfig, new SolverConfigContext(classLoader));
-    }
```

Comment:
Can we remove public API at this point?

Reply:
This wasn't released yet - so if we hurry, yes :)